postgres-exporter/Dockerfile

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2020 Astronomer Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# https://github.com/wrouesnel/postgres_exporter/blob/v0.8.0/Dockerfile
+FROM wrouesnel/postgres_exporter:v0.8.0 AS upstream
+# The above 'upstream' layer is FROM scratch (they use a multi-stage build)
+# I would like some normal tooling like package manager and shell available.
```

Comment:
I respectfully disagree with this. Containers should _not_ have any extra stuff in them that they don't need. If this is from scratch that is already ideal for a go binary.

Reply:
then how ought we to debug DB connection issue if we cannot exec into pod